testing/python/collect.py

```diff
@@ -1187,6 +1187,53 @@ def check3(fix1, i, fix2):
     )
     result = testdir.inline_run()
     result.assertoutcome(passed=6, failed=2)
+    
+    
+def test_collect_functools_partial_class(testdir):
+    """
+    Test that collection of functools.partial object works, and arguments
+    to the wrapped functions are dealt correctly (see #811).
```

Comment:
```suggestion
    to the wrapped functions are dealt correctly with (see #811).
```

Reply:
@blueyed see this PR for the original test case: https://github.com/pytest-dev/pytest/pull/5703